Assets/WizardsCode/Character/Documentation/uml.txt

```diff
@@ -0,0 +1,87 @@
+@startuml
```

Comment:
What is this file? Looks like it is a UML tool of some description? Is the file useful without the tool? If manually maintained I'd say not useful in the repo as it will be quickly out of date. Looks fairly incomplete even now.

Reply:
removed